### DISCUSSION TITLE
difference between oh my posh and oh my zsh

### DISCUSSION THREAD

**Discussion by terragady** [JanDeDobbeleer/oh-my-posh#3598] [General]
I am little confused as I found an article which suggest having them both and until now I thought they are the same tool for same stuff. Oh my zsh provide theming engine and ou can apply themes there including a powerful one which is p10k. Does it make sense to install on top of that an oh my posh? I have been using oh my posh in windows and a really like easiness of modifying a theme and better git status color control (pushed, not pushed etc). Does it make any sense to use both oh my zsh+oh my posh? Or choose just one to use?

**Comment by JanDeDobbeleer** [comment]
ohmyzsh offers a lot of additional features like autocompletion etc, so yes, you can use oh-my-posh alongside ohmyzsh only for the prompt configuration. That's also what I do. The name is historical, I agree it's somewhat confusing 😅

**Comment by terragady** [comment]
Oh OK that makes sense thanks 😁. How do you configure it? Which theme will be used? Or just don't set blank in oh my zsh?

**Reply by JanDeDobbeleer** [reply]
I import ohmyzsh and then initialise oh-my-posh with my own custom config.
